### DISCUSSION TITLE
Color and font.

### DISCUSSION THREAD

**Discussion by bringmetheaxx** [lauraramirez220612/TeamFourHYF#27] [General]
I tried to find the color and the font of the HYF and for the blue used the color finder says it's the #1C2294. 
The font the What Font Is? gave some option, but none of the seem to be the right one. 
What do you think about that? 

![color](https://user-images.githubusercontent.com/77135654/115233324-0a5baf80-a118-11eb-9828-abe90baccbc5.png)

![FONTS](https://user-images.githubusercontent.com/77135654/115232952-94574880-a117-11eb-98ce-7adeec86ffbf.png)

**Comment by Moamin-AbuEwaida** [comment]
@bringmetheaxx  I've just checked and fund the color is 4A50BB 
and the fonts for the paragraphs they used Space Mono and for the Headers Work Sans


![Space Mono](https://user-images.githubusercontent.com/28728040/115263104-6b45b080-a135-11eb-9a51-3be0e8becf7d.png)
![Work Sans](https://user-images.githubusercontent.com/28728040/115263124-70a2fb00-a135-11eb-8acd-70559e46febc.png)
![rgb(74, 80, 187)](https://user-images.githubusercontent.com/28728040/115263134-74368200-a135-11eb-8b1d-0c3675432934.png)

**Reply by danielhalasz** [reply]
yes, I have to agree, I have found the same, upon inspecting the DOM:
Space Mono, Work Sans, rgb(74, 80, 187)

**Reply by danielhalasz** [reply]
@bringmetheaxx have you tried using the dev tools?

**Comment by Moamin-AbuEwaida** [comment]
I want to add to this other colors codes
for the body text color is rgb (45, 48, 55)
the navbar links colors is rgb(127, 127, 127)
